cumulusci/cli/cci.py

```diff
@@ -786,15 +782,17 @@ def task_run(config, task_name, org, o, debug, debug_before, debug_after, no_pro
             import pdb
             pdb.set_trace()
 
-    except (TaskRequiresSalesforceOrg, TaskOptionsError) as e:
+    except CumulusCIUsageError as e:
         # Usage error; report with usage line and no traceback
         exception = click.UsageError(e.message)
         handle_exception_debug(config, debug, throw_exception=exception)
-    except (ApexTestException, BrowserTestFailure, MetadataComponentFailure,
-            MetadataApiError, ScratchOrgException) as e:
+    except CumulusCIFailure as e:
         # Expected failure; report without traceback
         exception = click.ClickException('Failed: {}'.format(e.__class__.__name__))
         handle_exception_debug(config, debug, throw_exception=exception)
+    except ScratchOrgException as e:
+        exception = click.ClickException('Failed: {}'.format(e.__class__.__name__))
+        handle_exception_debug(config, debug, throw_exception=exception)
```

Comment:
I don't understand your point about why ScratchOrgException should be handled separately. Currently you're handling it exactly the same as CumulusCIFailure and it's a subclass of CumulusCIFailure, so I don't think adding this block does anything.

Reply:
In fact, since the `except CumulusCIFailure` will match `ScratchOrgException` before getting to this block, I think this is unreachable code.

Reply:
Gah, what am I talking about -- ScratchOrgException extends CumulusCIException, not CumulusCIFailure. Sorry, need more ☕️...

Reply:
But I'm still not clear on why you don't want ScratchOrgException to be a CumulusCIFailure. And if it does need to be separate, you can still match both CumulusCIFailure and ScratchOrgException in the same `except` clause given that you're handling them in the same way.

Reply:
Okay, I get it.